runtime/compiler/env/j9method.h

```diff
@@ -112,11 +113,14 @@ class TR_J9MethodBase : public TR_Method
    bool                          isBigDecimalConvertersMethod( TR::Compilation * comp = NULL);
 
    static bool                   isUnsafeGetPutWithObjectArg(TR::RecognizedMethod rm);
+   static bool                   isTransformableUnsafeAtomic(TR::RecognizedMethod rm);
+   static TR::SymbolReferenceTable::CommonNonhelperSymbol equivalentAtomicIntrinsic(TR::RecognizedMethod rm);
```

Comment:
Are we planning to use these outside of UnsafeFastPath? If not I don't think they belong here as public APIs.